insta-unfollower.py

```diff
@@ -23,6 +23,20 @@
 
 session = requests.Session()
 
+
+class Credentials:
+    def __init__(self):
+        if os.environ.get('INSTA_USERNAME') and os.environ.get('INSTA_PASSWORD'):
+            self.username = os.environ.get('INSTA_USERNAME')
+            self.password = os.environ.get('INSTA_PASSWORD')
+        elif len(sys.argv) > 1:
+            self.username = sys.argv[1]
+            self.password = sys.argv[2]
+        else: 
+            sys.exit('Please provide INSTA_USERNAME and INSTA_PASSWORD environement variables or as an argument as such: ./insta-unfollower.py USERNAME PASSWORD.\nAborting...')
+
+credentials = Credentials() 
+
```

Comment:
Instead of creating weird function or long lines of condition I'd propose to simply make a credential class from which we can call the username and password by simply using `credentials.username` and `credentials.password` after creating an instance of Crendetials.

Reply:
ok, why not 👍🏼